driver/config/config.go

```diff
@@ -562,7 +562,7 @@ func (p *Config) CourierTemplatesRoot() string {
 }
 
 func (p *Config) parseURIOrFail(key string) *url.URL {
-	u, err := url.ParseRequestURI(p.p.String(key))
+	u, err := url.Parse(p.p.String(key))
```

Comment:
Looks good! I think we should still check if there are the following things set:

- scheme (`http(s)://`)
- host

as `url.Parse` only returns an error if the string is empty!

Reply:
The host is `""` on absolute file URLs btw